File: tests/benchmark/fib.lua
function fib(n)
    if n < 2 then
        return n
    else
        return fib(n - 1) + fib(n - 2)
    end
end

clock = os.clock
time_start = clock()
print(fib(35) == 9227465)
time_end = clock()

print("elapsed:")
print(time_end - time_start)

File: tests/benchmark/fib.py
from __future__ import print_function

import time

def fib(n):
    if n < 2: return n
    return fib(n - 1) + fib(n - 2)

start = time.process_time()
print(fib(35) == 9227465)
end = time.process_time()

print("elapsed:")
print(str(end - start))